Documentation/Books/Manual/Architecture/DeploymentModes/ActiveFailover/Architecture.md

```diff
@@ -45,12 +45,20 @@ When the _Leader_ goes down, this is automatically detected by the _Agency_
 instance, which is also started in this mode. This instance will make the
 previous follower stop its replication and make it the new _Leader_.
 
-The _Follower_ will deny all read and write requests from client applications.
+Operative Behaviour
+-------------------
+
+In contrast to the normal behaviour of a single-server instance, the Active-Failover
+mode will change the behaviour of ArangoDB in some situations.
+
+The _Follower_ will _always_ deny write requests from client applications. Read requests
```

Comment:
please add the version from when this is possible, e.g. "starting from version XYZ" and/or "before version XYZ this was not possible and this other was happening"